Update README.md to let users know to use flexible server instead. (#107)
hmcts/cnp-module-postgres: README.md

> # Do not use this module of you are creating a new database / new team starting to deploy etc.

Reviewer: ```suggestion
# **Warning** Deprecated, do not use for new services.
```

> All new teams to deploy Postgres Flexible server using this module [here](https://github.com/hmcts/terraform-module-postgresql-flexible). 

Reviewer: ```suggestion
All new teams are to deploy [Postgres Flexible server](https://github.com/hmcts/terraform-module-postgresql-flexible). 
```

> This module will be moved into read-only/deprecated once teams have migrated to Flexible server.

Reviewer: ```suggestion
This module will be archived once all teams have migrated to Flexible server.
```
